site/.babelrc.js

```diff
@@ -1,23 +0,0 @@
-module.exports = {
-  presets: [
-    [
-      "next/babel",
-      {
-        "preset-env": {
-          targets: {
-            chrome: "73",
-          },
-          // https://github.com/vercel/next.js/issues/17273#issuecomment-700700214
-          // Can be removed after upgrading to Next 12
```

Comment:
🎉

We don’t need any custom babel plugis, so can rely on default `swc` parser now!